### ISSUE TITLE
Never unsuspend this site after search

### ISSUE THREAD

**Issue by MonsterNya** [deanoemcke/thegreatsuspender#839]
This site (search page) (http://kinozal.tv/browse.php) can not wake up after suspend. 
Not work after search russian text (кириллица).

**upd**: not work on any site. when the search query is filled in url

normal, search word "test"
chrome-extension://klbibkeccnjlkjkiokjodocebajanakg/suspended.html#ttl=%D0%A0%D0%B0%D0%B7%D0%B4%D0%B0%D1%87%D0%B8%20%3A%3A%20%D0%9A%D0%B8%D0%BD%D0%BE%D0%B7%D0%B0%D0%BB.%D0%A2%D0%92&pos=0&uri=http://kinozal.tv/browse.php?s=test
![image](https://user-images.githubusercontent.com/23438485/50585947-73a44f80-0e88-11e9-9547-1a53795e8f2c.png)

abnormal, search word "тест"
chrome-extension://klbibkeccnjlkjkiokjodocebajanakg/suspended.html#ttl=%D0%A0%D0%B0%D0%B7%D0%B4%D0%B0%D1%87%D0%B8%20%3A%3A%20%D0%9A%D0%B8%D0%BD%D0%BE%D0%B7%D0%B0%D0%BB.%D0%A2%D0%92&pos=0&uri=http://kinozal.tv/browse.php?s=%F2%E5%F1%F2&g=0&c=0&v=0&d=0&w=0&t=0&f=0
![image](https://user-images.githubusercontent.com/23438485/50585919-4a83bf00-0e88-11e9-98e2-76c88cf823bf.png)

 * Extension version: 7.0.109
 * Browser name & version: chrome 71.0.3578.98
 * Operating system & version: win10 x64 1703

**Comment by deanoemcke** [comment]
Sorry but I cannot replicate this issue. If you are familiar with installing extensions from source, can you try with the latest development version of the extension? 
The latest is v7.0.159.

**Comment by Jollfye** [comment]
159 tried, also can't replicate the issue. Tab with russian text in url unsuspends as usual, with click almost anywhere in it.

**Comment by MonsterNya** [comment]
**Jollfye**
you are using version 7.0.109 or newer?

**deanoemcke**
v7.0-beta.128 latest version which i see

**Comment by Jollfye** [comment]
@MonsterNya I'm using 159 now as I wrote above and here's how u can too as I assume u don't know how to use the latest development version: go to TGS github main page here https://github.com/deanoemcke/thegreatsuspender , green button clone or download , download zip (direct url https://github.com/deanoemcke/thegreatsuspender/archive/master.zip , it is the latest 160 now version, latest commit from our ram savior, developer dear @deanoemcke ), unpack zip anywhere u'd want it to stay (like maybe into ur Chrome browser folder), then in TGS settings, make a screenshot of it, copy ur never suspend urls list to somewhere (to save it if u have it), then after, in session management, export the current session, also to save it externally, and here u can remove ur TGS Chrome Web Store version (which is usually far behind latest, for reasons) from Chrome Extensions, then enable developer mode and load unpacked TGS extension from previously unzipped folder thegreatsuspender-master\src , exactly from src, that's where extension really is, recover ur settings, import ur session aand I guess that's it, u r on latest version. From now on to update extension u will have to manually download zip from time to time and unpack it to it's 'stay' folder replacing files while Chrome is preferably exited. Also I think there may be another way, without removing current Web Store version, to find TGS extension folder in ur Chrome profile (usually somewhere here C:\Users\<User>\AppData\Local\Google\Chrome\User Data\Default\Extensions with the name of the folder being that line of letters from any of ur suspended tab's url), then in that folder I assume there's may be another folder with the Web Store TGS extension version name, something 109 for now, exit Chrome, delete everything from this 109 folder, copy/paste to it all files from zip src, start Chrome and I think TGS should get updated here, but I'm not sure, will it even work? And how it will get along with Web Store, maybe it will tell u extension was 'corrupted' aka wanna repair (reinstall)? I didn't use this method but feel free to try, of course with backups ;) Oh n updating here also will be the same manual files replacing from zips but from just src folder. But again idk how it will be with the Web Store, how it updates then. Uh anyway, good luck, check ur issue with the latest version and let the dev kno if u do ^_^

**Comment by MonsterNya** [comment]
@Jollfye @deanoemcke 
Oh, thank you for a truly comprehensive answer. I once tried to make backup copies of chrome extensions, but the settings remained the same after moving to a folder, as if I did not copy anything.
I installed the extension version 160, through developer mode.
the bug I described is already missing, everything works fine.

One more question, the extension was installed in Russian language, I did not find in the settings how to switch to English? Can I recommend adding a switch? Many Russian users will want to continue using the English version.

I would also like to inform you that the Russian translation contains inaccuracies and errors, can I offer corrections to the translation somewhere?

**Comment by Jollfye** [comment]
@MonsterNya backing up Chrome extensions may be tricky coz C:\Users\User\AppData\Local\Google\Chrome\User Data\Default\Extensions folder I think only stores static files for Chrome Web Store extensions (not their settings, just extensions themself), and where their settings then? I found some answers here https://superuser.com/questions/942392/how-can-i-modify-data-stored-by-chrome-extensions . So when I searched through files in Default folder with the TGS folder name from any suspended url, for me it is gbcokmhikpgllchhjdjdklejaicdbhmg, it found 2 folders: C:\Users\User\AppData\Local\Google\Chrome\User Data\Default\IndexedDB\chrome-extension_gbcokmhikpgllchhjdjdklejaicdbhmg_0.indexeddb.leveldb and C:\Users\User\AppData\Local\Google\Chrome\User Data\Default\Sync Extension Settings\gbcokmhikpgllchhjdjdklejaicdbhmg , so I guess that's where TGS stores it's settings, but even if u copy those files and paste/replace them later with new TGS installation, they may be overwritten when Chrome starts so no settings will be gotten from them. Soo how to backup settings of a particular extension and then recover them successfully I myself don't know yet, needs more googling I guess.. But how to save all Extensions, their settings as well as all Chrome settings and profile, yep, just copy Default folder and that's it.

And I'm glad the problem u had is now gone, so dev can close this issue 😊

I think TGS gets language setting from chrome://settings/ Languages, tho switch is a good idea and u could probably go create new issue (feature request) for TGS here on github, for now u could change ur entire Chrome browser language to English (United States), I guess that may be not such a big problem for u as u know English to some extent obviously and that's if my assumption about where TGS gets its language setting is correct.

About translation, the original English is here https://github.com/deanoemcke/thegreatsuspender/blob/master/src/_locales/en/messages.json and russian is here https://github.com/deanoemcke/thegreatsuspender/blob/master/src/_locales/ru/messages.json , press the pencil to fork and edit the file then u could create a new pull request with ur corrections for dev and everybody else to comment and review. Have fun, it would be nice to see ur corrections 😉
